test/quantization/test_quant_api.py

```diff
@@ -493,7 +513,7 @@ def test_quantized_tensor_subclass_int8(self):
     @unittest.skipIf(not torch.cuda.is_available(), "Need CUDA available")
     def test_quantized_tensor_subclass_int8_dyn_quant(self):
         # use 1024 so that we don't need padding
```

Comment:
nit: comment needs updating, how come this was changed?

Reply:
oh I was testing around transpose related things before, I can update the comment, it's good to have different out/in dimensions for weight to make sure we also test the transpose I think